lte/gateway/python/magma/enodebd/devices/freedomfi_one.py

```diff
@@ -233,7 +233,8 @@ def set_magma_device_cfg(
         success_str = "SUCCESS"  # String constant returned by radio
         insync_str = "INSYNC"
 
-        if name_to_val[cls.DEFAULT_GW].upper() != success_str:
+        if name_to_val[cls.DEFAULT_GW] \
+                and name_to_val[cls.DEFAULT_GW].upper() != success_str:
```

Comment:
One potential alternative to also do the lookup (and also avoid any `KeyError` exception) it's
```suggestion
        if name_to_val.get(cls.DEFAULT_GW, "").upper() != success_str:
```

However I'm not sure how often could we hit the said exception here (given on how this dictionary is constructed), so I'm fine with the current way of handling this.